frontend/pages/hosts/details/cards/Software/HostSoftware.tsx

```diff
@@ -245,6 +296,14 @@ const HostSoftware = ({
     ]
   );
 
+  const getHostSoftwareFilterFromQueryParams = useCallback(() => {
+    const { available_for_install } = queryParams;
+    if (available_for_install) {
+      return "installableSoftware";
+    }
+    return "allSoftware";
```

Comment:
```suggestion
    return queryParams.available_for_install ? "installableSoftware" : "allSoftware";
```

Non-blocking, but the one-liner seems clear enough?

Reply:
💯